.github/workflows/bazel_test.yml

```diff
@@ -31,3 +35,4 @@ jobs:
         # The "<fuzzer/FDP...>" is only supported since clang-10 while the clang version in github runner is 9
         run: |
           bazel build --verbose_failures --build_tag_filters=fuzz_test,-FDP --config=asan-libfuzzer //examples/...
+          bazel run //examples:empty_fuzz_test_run --config=asan-libfuzzer -- --timeout_secs=5
```

Comment:
What version of python is used when running _run?

Reply:
The `python` command in github runner is python2. I have to declare `alias python="python3"` to run python3